cfpb/capital-framework
# Atomic/typography

Converted cf-typography to atomic standards

## Removals

- Removed custom bullet mixin because it is no longer used
- Removed fancy slug because it is no longer used
- Removed short-desc because we are now using Avenir as our standard body font
- Removed padded heading because it's no longer used.
- Removed icon lists for the time being to be addressed later when we can look at existing implementations.

## Changes

- Broke links apart to create less repetitive code
- Broke headings and headers apart to create a better parent/child relationship and handle layout according to our best practices.

## Testing

- check out `example/cf-typography ` sandbox branch and run `npm run build && npm start`
- navigate to `http://localhost:3000/components/cf-typography/`

## Review

- @Scotchester 

## Screenshots

Too many to list, but everything should really look the same as the examples at `https://cfpb.github.io/capital-framework/components/cf-typography/#pull-quote` 

## Notes

- `a-heading` is a placeholder much like `a-list`. There are no uniform styles between the headings, but it makes the most sense, especially when controlling the layout/margin of stylized headings nested inside headers.
- __Reduces output by 4kb__. Small gains but still a positive impact.

## Todos

- Clean up the way we're handling icons. There's a ton of repetitive code due to icons and I have an idea of how it could be better implemented.

## Checklist
